pkg/restic/common.go

```diff
@@ -79,6 +69,7 @@ func GetPodSnapshotAnnotations(obj metav1.Object) map[string]string {
 
 // SetPodSnapshotAnnotation adds an annotation to a pod to indicate that
 // the specified volume has a restic snapshot with the provided id.
+// Deprecated: we will stop using pod annotations to record restic snapshot IDs after they're taken.
 func SetPodSnapshotAnnotation(obj metav1.Object, volumeName, snapshotID string) {
```

Comment:
we should be able to delete this function now, since we shouldn't be using it anywhere anymore.